### DISCUSSION TITLE
DB port 21699 breaks Database Connection

### DISCUSSION THREAD

**Discussion by BernhardFuchs** [requarks/wiki#4751] [Error / Bug Report]
Currently we run the wiki with Docker in a Kubernetes cluster (not Helm) and with Postgres installed on a separate VM on Port 5432.  
Now we are migrating the Database to a DBaaS solution which runs Postgres on Port 21699 (not configurable).  
When we use the new host/port combination, following error is thrown:
```
error: Database Connection Error: 28000 undefined:undefined
```

Using just the new port results in a connection timeout which is expected. In this case it seems that the env variables are passed correct
```
error: Database Connection Error: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
```

I did a local test with intentional Auth error and get following logs with using IP address and 4 digit Port 5432
```
error: Database Connection Error: ECONNREFUSED xxx.xxx.xxx.xxx:5432
```
and the same strange error with same IP address and 5 digit Port 21699
```
error: Database Connection Error: 28000 undefined:undefined
```

To me it seems that the app cannot handle port numbers of a certain range. Not sure if it is related to the number of digits.

**Comment by aidabeorn** [comment]
the number of digits in the database's port isn't an issue, I'm using DBaas as well and have no problems, my instance uses 33882.
